feature-group-definitions/array-group.yml

```diff
@@ -1,4 +1,13 @@
 spec: https://tc39.es/proposal-array-grouping/
+status:
+  baseline: false
+  support:
+    chrome: "117"
+    chrome_android: "117"
+    edge: "117"
+    firefox: "119"
+    firefox_android: "119"
+    safari: preview
```

Comment:
Do we really need this? It would be nice if this is always a version string or missing, so that consumers don't need to check.

Reply:
No, we don't. Someday we might want to have a "upcoming" status, where we may reconsider this. But for now, I'll update my tool to hide future-dated and preview releases, then update this PR.